client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/sidebar.tsx

```diff
@@ -212,7 +212,7 @@ const Sidebar = ( { sidebarDomain, siteSlug, submit, goNext, goToStep, flow }: S
 				) }
 				<Launchpad
 					siteSlug={ siteSlug }
-					taskFilter={ () => enhancedTasks }
+					taskFilter={ () => enhancedTasks || [] }
```

Comment:
Why is this change needed as part of this PR?

Reply:
It is not directly related to the changes described in the PR. It's a small refactor in the spirit of _a leave-the-code-better-than-you-found-it_ kind of change.
I thought making the filter always return a `Task[]` would make the filter interface cleaner and easier to debug. Having to add `|| []` was a side effect.